src/main/java/racingcar/domain/RacingCars.java

```diff
@@ -0,0 +1,74 @@
+package racingcar.domain;
+import racingcar.strategy.DoOneForward;
+import racingcar.strategy.OneForwardCondition;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * 일급 콜렉션, 콜렉션 wrapping하며 그 콜렉션 외에 다른 변수는 없는 상태
+ * 해당 콜렉션이 아닌 단순 배열/리스트는 ..List ...Arr 로 해서 헷갈리지 않게
+ */
+
+public class RacingCars {
+
+    private List<Car> cars;
+    private int racingCounts;
+
+    public List<Car> getCars() {  //unModifiable acc. to javajigi
+        return Collections.unmodifiableList(cars);
+    }
+
+    public RacingCars(int carCounts, int racingCounts) {
+        this.cars = createCars(carCounts);
+        this.racingCounts = racingCounts;
+    }
+
+    public static ArrayList<Car> createCars(int carCounts) {
+        ArrayList<Car> cars = new ArrayList<>(carCounts);
```

Comment:
ArrayList는 List 인터페이스를 상속받은 클래스로 보통 List 로 타입을 지정합니다 😄 
```suggestion
        List<Car> cars = new ArrayList<>(carCounts);
```